### PR TITLE
 Typescript: Add locale to MuiPickersUtilsProviderProps

### PR DESCRIPTION
**Pull request by tstaehli**
- [x] I have changed my target branch to **develop** :facepunch:

# 

## Description
very small change; simply add the newly introduced locale prop of MuiPickersUtilsProviderProps in the corresponsinf typing file

### CODE REVIEW

**Comment by dmtrKovalenko** [comment]
@tstaehli build failed :(

File: lib/src/utils/MuiPickersUtilsProvider.d.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { Utils } from '../typings/utils';
 export interface MuiPickersUtilsProviderProps {
   utils: Utils;
   children: ReactNode;
+  locale: (any | string);
```
Comment (dmtrKovalenko) [inline lib/src/utils/MuiPickersUtilsProvider.d.ts]: It should not be required

**Comment by tstaehli** [comment]
changes applied. Not sure about the build though; this does not seem like a change that could break the build...

**Comment by dmtrKovalenko** [comment]
@tstaehli we have typescript usage tests. And there is no locale passed — that’s why the build failed

**Comment by dmtrKovalenko** [comment]
@tstaehli btw why do you use any | string - is it really needed.  Any will cover string as well

**Comment by tstaehli** [comment]
@dmtrKovalenko ah, that explains the test failure. good testsuite then :-) I adapted the type from the MuiPickersUtilsProvider.jsx where the prop is defined as `PropTypes.oneOfType([PropTypes.object, PropTypes.string]),`.
I realise that `any` would suffice; however I wanted to emphasize that locale is either a string for use with MomentUtils and a more complex for DateFnsUtils.

**Comment by tstaehli** [comment]
sorry, wrong button

**Comment by cescoferraro** [comment]
@dmtrKovalenko please merge!

**Comment by cescoferraro** [comment]
would you mind making another release?? @dmtrKovalenko thanks!

**Pull request merged**